tox.ini

```diff
@@ -0,0 +1,27 @@
+[tox]
+envlist = lint,py36
+
+[testenv]
+deps =
+    -rdev-requirements.txt
+whitelist_externals =
+    make
+    touch
+setenv =
+	CACHITO_SETTINGS={toxinidir}/settings.cfg
+commands =
+    touch {toxinidir}/settings.cfg
+    py.test tests/ {posargs}
+
+[testenv:lint]
+deps =
+    flake8
+commands =
+    python -m flake8 {posargs}
+
+[flake8]
+show-source = True
+max-line-length = 100
+exclude = .git,.tox,dist,*egg,*fedmsg.d
```

Comment:
Oh no, not fedmsg!

Reply:
Honestly, I don't know how that got in there. Removed.